UKHomeOffice/nmsw-ui
# NMSW-1343 Link to passenger question when it's a departure from UK journey

# Ticket



----

## AC

Given I am viewing the task list page for a departure from the UK journey
When I click the link to passengers
And AMG processing is completed (no checkins requested, no errors)
Then I am taken to the 'Do you have passengers on this journey' page

----

## To test

- create a voyage with a departure from UK
- upload a FAL6 and wait until all people are checked in
- > task details should still link to 'are there passengers' question
- create a voyage with an arrival to the UK
- upload a FAL6 and wait until all people are checked in
- > task details should link to the UPT results page

----

## Notes
Extended tests on this link to cover more scenarios, and moved the tests to their own file so they're easier to read
